### ISSUE TITLE
Suggestion to add C57BL/6NTac

### ISSUE THREAD

**Issue by kristenpeach** [EBISPOT/efo#1930]
For new term requests, please provide the following information:

## Preferred term label

C57BL/6NTac

## Synonyms

C57BL/6Tac

## Textual definition

C57BL/6NTac inbred strain of Mus musculus. MGI ID: 2164831 https://www.informatics.jax.org/strain/MGI:2164831

## Suggested parent term

C57BL http://www.ebi.ac.uk/efo/EFO_0005181

## Attribution

https://orcid.org/0000-0002-6388-0901

**Comment by kristenpeach** [comment]
@PaulNSchofield

**Comment by zoependlington** [comment]
Term added as EFO_0020093, which will be available after our April release on the 17th.
